src/sonic-host-services/scripts/caclmgrd

```diff
@@ -215,18 +216,12 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
                 for key, _ in iface_table.items():
                     if not _ip_prefix_in_key(key):
                         continue
-
                     iface_name, iface_cidr = key
-                    ip_ntwrk = ipaddress.ip_network(iface_cidr, strict=False)
-
-                    # For VLAN interfaces, the IP address we want to block is the default gateway (i.e.,
```

Comment:
Looks like the comment is to block default g/w and the code change here is intended.

Reply:
Blocking the default GW in the INPUT table doesn't make sense -- unless you are the default GW in which case my code does the exact same thing.

Reply:
ok